docs/solutions/sql_server_always_on_availability_groups/README.md

```diff
@@ -1 +1,35 @@
-## Coming Soon!
+Solution: MS SQL Always On.
+
+This Terraform template implements the infrastructure needed to fully deploy a cross AD, MS SQL Always On cluster.  The cluster can be spread across either 2 or 3 ADs, depending on your needs/preferences, but cannot be located in a single AD.
+
+The template is configured by modifying the variables contained in two files:
+
+configuration.tf - generalized configuration for the environment as a whole.
+sql.tf - configuration specific to the implementation of MS SQL Always On.
+
+To configure, simply edit these two files, apply values that are appropriate to your environment, and run 'terraform plan; terraform apply'.  Before attempting to run the template, setup your environment by performing the following:
+
+- Edit the env-vars file.
+- Set the values indicated in the file
+- Source the file into your existing environment by running:
+
+. ./env-vars
+
+Some items of note:
+
+- This has only been tested using Windows 2012 R2 images.  The template and accompanying whitepaper has not been deployed using Windows 2016 Datacenter within the OCI environment.
+- The image OCID listed in the configuration.tf file is the one used for testing.  You should update this with the image OCID of the latest revision of Windows 2012 R2.
```

Comment:
Might want to link to the image OCID list:
https://docs.us-phoenix-1.oraclecloud.com/Content/Resources/Assets/OracleProvidedImageOCIDs.pdf

Reply:
Done